src/LiveComponent/src/LiveComponentHydrator.php

```diff
@@ -392,7 +394,14 @@ private function dehydrateObjectValue(object $value, string $classType, ?string
             }
         }
 
-        throw new \LogicException(sprintf('Unable to dehydrate value of type "%s" for property "%s" on component "%s". Either (1) change this to a simpler value, (2) add the hydrateWith/dehydrateWith options to LiveProp or (3) set "useSerializerForHydration: true" on the LiveProp.', $value::class, $propertyPathForError, $componentClassForError));
+        $hydratedObject = [];
+        foreach ((new \ReflectionClass($classType))->getProperties() as $property) {
```

Comment:
I think we should use https://symfony.com/doc/current/components/property_info.html#list-information instead of `ReflectionClass::getProperties()`. It'll let us also support private properties with getters & setters (I believe the property info will only return public properties + private properties with a getter).

Reply:
I think this isn't resolved yet. We're using processAccessor to read the values, but not property info to list the properties.